pkg/client/client.go

```diff
@@ -573,6 +573,11 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 			protocols = strings.Join(hs.Protocols, ", ")
 		}
 
+		gracefulTerm := "Enabled"
+		if !sr.KubeProxyReplacement.Features.GracefulTermination.Enabled {
+			gracefulTerm = "Disabled"
+		}
```

Comment:
Just tiny nit: for most other features above we start out with `"Disabled"` and then when the feature is enabled, we also set the string to `"Enabled"`. Why going the other way round here?

Reply:
Well, I flipped the status string because this feature is enabled by default. I don't have a preference either way. I can keep the assignment consistent.